zerver/models.py

```diff
@@ -571,6 +571,11 @@ class Stream(ModelReprMixin, models.Model):
         max_length=32, default=generate_email_token_for_stream) # type: text_type
     description = models.CharField(max_length=1024, default=u'') # type: text_type
 
+    # Specify default permissions assigned when user subscribes to a stream.
+    PERMISSION_FLAGS = ['can_read', 'can_write', 'can_moderate',
```

Comment:
Since `Stream` and `Subscription` both share the same flags, you probably only want one `PERMISSION_FLAGS` variable that's shared between both classes.

Some options off the top of my head include making `PERMISSION_FLAGS` global, or keeping it in one class (for example `Stream`) and referring to the class variable from `Subscription`. The second option seems better to me